src/Controls/samples/Controls.Sample/Pages/Core/ContextFlyoutPage.xaml

```diff
@@ -0,0 +1,128 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage 
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    x:Class="Maui.Controls.Sample.Pages.ContextFlyoutPage">
+
+    <ScrollView>
+        <VerticalStackLayout>
+            <HorizontalStackLayout Spacing="10">
+                <Button Text="Increment by 1 (or right-click me)" Clicked="OnIncrementByOneClicked" FontSize="30" BackgroundColor="DarkGray">
+                    <FlyoutBase.ContextFlyout>
+                        <MenuFlyout>
+                            <MenuFlyoutItem Text="Increment by 10" Clicked="OnIncrementMenuItemClicked" CommandParameter="10"></MenuFlyoutItem>
```

Comment:
`MenuFlyoutItem` inherits from MenuItem and there are inherited properties that don't work. For example, using **IsEnabled** to disable a MenuFlyoutItem doesn't work.

Reply:
Yea, we can log a bug on this for now and look at fixing after we're past code freeze

Reply:
https://github.com/dotnet/maui/issues/9583